weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1833,6 +1833,75 @@ def feedback_create(self, req: tsi.FeedbackCreateReq) -> tsi.FeedbackCreateRes:
             payload=res_payload,
         )
 
+    def feedback_create_batch(
+        self, req: tsi.FeedbackCreateBatchReq
+    ) -> tsi.FeedbackCreateBatchRes:
+        """Create multiple feedback items in a batch efficiently."""
+        results = []
+        rows_to_insert = []
+
+        print('>>>>', len(req.batch))
+
+        for feedback_req in req.batch:
+            assert_non_null_wb_user_id(feedback_req)
+            validate_feedback_create_req(feedback_req, self)
+
+            # Augment emoji with alias (same logic as individual create)
```

Comment:
what does this mean?

Reply:
ill DRY with above, this was just copied from that. will get the magic number too